src/docfx/build/metadata/UserMetadata.cs

```diff
@@ -21,6 +21,9 @@ internal class UserMetadata
 
         public SourceInfo<string> Author { get; private set; } = new SourceInfo<string>("");
 
+        [JsonProperty("ms.author")]
+        public SourceInfo<string> MsAuthor { get; private set; } = new SourceInfo<string>("");
```

Comment:
make `ms.author` nullable so if a file does not have `ms.author`, it should not appear in `.errors.log`. Update test case to assert `ms.author` does not appear using `"ms.author": undefined"`
```suggestion
        public SourceInfo<string?> MsAuthor { get; private set; } = new SourceInfo<string?>();
```